spec/spec.md

```diff
@@ -499,6 +490,31 @@ Below is a non-normative example of a decoded VC in a JSON format, signed as a J
 ```
 :::
 
+#### [[def:Authentication Response Validation]]
+
+The following checks MUST be made by the Verifier upon receiving the Authorization Response:
+
+##### ID Token Validation
+
+To validate the ID Token received, the Verifier MUST do the following:
+
+1. The Verifier MUST validate ID Token according to the rules defined in section 12.1 of [[ref: SIOPv2 ID1]].
+2. The ID Token MUST be signed by the End-User's DID:
+3. `iss` claim MUST be `https://self-issued.me/v2/openid-vc`.
+4. The signature on the ID Token MUST be validated. Validation is performed against the key obtained from a DID Document. DID Document MUST be obtained by resolving a Decentralized Identifier included in the `sub` claim using DID Resolution.
+5. `sub` claim MUST equal the `id` property in the DID Document.
+6. The `_vp_token` claim MUST be present and contain a presentation_submission with a valid descriptor map.
+
+##### VP Token Validation
+
+To Validate the VP Token received, the Verifier MUST do the following:
+
+1. The signature on the VP Token MUST be validated. Validation is performed against the key obtained from a DID Document. DID Document MUST be obtained by resolving a Decentralized Identifier included in the `sub` claim using DID Resolution.
+2. Using the descriptor map defined the the ID Token, the Verifier MUST obtain the VPs and validate their integrity, authenticity, and holder binding. Inputs should be evaluated using the mechanisms outlined in Section 4.3 of [[ref: Presentation Exchange v1.0.0]].
+3. The revocation status of the VCs embedded in the VPs MUST match those requested in the Presentation Definition of the Authorization Request. The Verifier MUST check the expected status by discovering the relevant Status List using either DID Relative URLs stored in an HTTPS URL or ID Hubs.
```

Comment:
```suggestion
3. The Verifier MUST validate the signature of the VC(s) embedded in the VPs. <needs more text>
4. The Verifier MUST check the status of VC(s) by discovering the Status List using either DID Relative URLs stored in an ID Hub or an HTTPS URL. 
```
I do not believe the profile currently mandates requesting credential of a specific status. so we cannot require matching those at this moment.

A step to validate VC signatures should be separate,

Reply:
i could have sworn we talked about this, but i think i was mixed up with something else... i will change to recommend that you check the revocation mentioned presentation definition if its there.